x-pack/plugins/observability/public/components/shared/exploratory_view/obsv_exploratory_view.tsx

```diff
@@ -91,7 +95,7 @@ export const obsvReportConfigMap = {
     getMobileKPIDistributionConfig,
     getMobileDeviceDistributionConfig,
   ],
-  [DataTypes.METRICS]: [getMetricsKPIConfig],
+  [DataTypes.LOGS]: [getLogsKPIConfig],
```

Comment:
Good catch, I’ll take a look 👍

Reply:
had the issue when i tried to change the report type

Reply:
I've fixed this, although it's not an ideal fix ☹️ 

There's a hardcoded list of exceptions that look like this:

```
const options = dataTypes
    .filter(({ id }) => {
      if (reportType === ReportTypes.DEVICE_DISTRIBUTION) {
        return id === DataTypes.MOBILE;
      }
      if (reportType === ReportTypes.CORE_WEB_VITAL) {
        return id === DataTypes.UX;
      }
      return true;
    })
    .map(({ id, label }) => ({
      value: id,
      inputDisplay: label,
    }));
```

This is why device distribution and core web vitals didn't break.

The nicer, and generic, way of doing this would be to say "does this data type have a config registered with the matching report type?", however this component only has access to the `reportConfigMap` via it's context / state, the `reportConfigMap` only gives us the config generation functions, and at this level in the hierarchy I can't see a convenient way to gain access to the parameters the functions require to call them (the generated `dataView` and `series`).

As such I have just added a hard coded Logs exception.